### PR TITLE
Add missing file licences and do some clean works.

### PR DESCRIPTION
**Pull request by yanggangtony**
Thank you for contributing to Velero!

# Please add a summary of your change
1 add missing file licences
2 fix docs error typo in ```volume-snapshot-data-movement.md```
3 optimize the logs in ```download_request_controller.go``` 

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/controller/download_request_controller.go [hunk 1]
```diff
@@ -169,10 +169,10 @@ func (r *downloadRequestReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			Name:      backup.Spec.StorageLocation,
 		}, location); err != nil {
 			if apierrors.IsNotFound(err) {
-				log.Errorf("BSL for DownloadRequest cannot be found")
+				log.Errorf("fail to get BSL for DownloadRequest")
```
Comment (mateusoliveira43) [inline pkg/controller/download_request_controller.go:172]: here isn't better to have a different message from below how it was?

Comment (yanggangtony) [reply pkg/controller/download_request_controller.go:172]: mistake, thanks review.

**Review by sseago — APPROVED** [review]

**Comment by ywk253100** [comment]
@yanggangtony Please check the CI failure

**Comment by yanggangtony** [comment]
> @yanggangtony Please check the CI failure

Oh, i misup the test code , when continue the schedule is ''
I will fix it later.

**Review by ywk253100 — DISMISSED** [review]

File: internal/resourcemodifiers/resource_modifiers.go [hunk 2]
```diff
@@ -1,3 +1,18 @@
+/*
+Copyright The Velero Contributors.
```
Comment (kaovilai) [inline internal/resourcemodifiers/resource_modifiers.go:2]: According to https://www.apache.org/licenses/LICENSE-2.0#apply

Year is required. Thoughts?
```
Copyright [yyyy] [name of copyright owner]
```

Comment (yanggangtony) [reply internal/resourcemodifiers/resource_modifiers.go:2]: I just like the license can be forever.

Comment (yanggangtony) [reply internal/resourcemodifiers/resource_modifiers.go]: Rebased , add the years.

Comment (sseago) [reply internal/resourcemodifiers/resource_modifiers.go]: Actually, according to the project code standards, copyright year is to be omitted.
https://velero.io/docs/v1.12/code-standards/#copyright-header

Comment (sseago) [reply internal/resourcemodifiers/resource_modifiers.go]: I'm not sure what led to the code standards decision to remove the year. It probably should be there, but the fix to that is to change the code standards (a separate issue from this).

Comment (yanggangtony) [reply internal/resourcemodifiers/resource_modifiers.go]: yes, so how to handle this case , as before , i delete the year? WDYT@sseago

Comment (yanggangtony) [reply internal/resourcemodifiers/resource_modifiers.go:2]: Follow the code standards guide, delete the year..🥲🥲

**Review by sseago — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**